compose.yaml

```diff
@@ -20,7 +20,7 @@ services:
     image: "${OLIMAGE:-oldev:latest}"
     environment:
       - OL_CONFIG=${OL_CONFIG:-/openlibrary/conf/openlibrary.yml}
-      - GUNICORN_OPTS=${GUNICORN_OPTS:- --reload --workers 1 --timeout 180 --max-requests 500}
+      - GUNICORN_OPTS=${GUNICORN_OPTS:- --reload --workers 2 --timeout 180 --max-requests 500}
```

Comment:
Note we'll want to apply the same to compose.production.yml if we want this to affect prod ; and that will require some careful testing, since this did cause some pressure on the prod servers when we had it to a larger number -- 45 :P So this might be best in its own PR so we can test on prod before merging.